macros/calibration/calibrate.cfg

```diff
@@ -3,23 +3,26 @@ description: Calibrate the printer flow or pressure advance
 gcode:
     # Type of calibration
     {% set TYPE = params.TYPE|default("")|string|lower %}
-    # Define vars with safe default values for ABS (main material I'm using)
     {% set BED_TEMP = params.BED_TEMP|default(printer["gcode_macro _USER_VARIABLES"].print_default_bed_temp)|float %}
     {% set EXTRUDER_TEMP = params.EXTRUDER_TEMP|default(printer["gcode_macro _USER_VARIABLES"].print_default_extruder_temp)|float %}
-    {% set MATERIAL = params.MATERIAL|default("ABS")|string %}
+    {% set MATERIAL = params.MATERIAL|default(printer["gcode_macro _USER_VARIABLES"].print_default_material)|string %}
+    {% set CONTROL_SPEED = params.CONTROL_SPEED|default(80)|string %}
+    {% set TRAVEL_SPEED = params.TRAVEL_SPEED|default(100)|string %}
+	{% set DO_RAFT = params.DO_RAFT|default(1)|string %}
+    {% set RAFT_SPEED = params.RAFT_SPEED|default(80)|string %}

```

Comment:
Not sure about this one. I made this file only to provide an example of how to call the calibration macros, but not really like a generic way to call them properly. @elpopo-eng what do you think about that ?

Maybe, a good compromize would be to remove possibility to add parameters to the call of `CALIBRATE`. But keep the parameters of the calibrations in the `variables.cfg`. So that way, everyone can set some custom values in their variables file that are then used when calling `CALIBRATE TYPE=flow/pressure_advance`.
What do you think about that ?

Reply:
Sounds good